external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/KafkaSpout.java

```diff
@@ -73,7 +73,9 @@
     // Strategy to determine the fetch offset of the first realized by the spout upon activation
     private transient FirstPollOffsetStrategy firstPollOffsetStrategy;  
     // Class that has the logic to handle tuple failure
-    private transient KafkaSpoutRetryService retryService;              
+    private transient KafkaSpoutRetryService retryService;
+    // Handles the events off a tuple
```

Comment:
Nit: Rephrase as "Handles tuple events (emit, ack etc.)", I don't think it is clear what this means.